src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person
+  people: Person[],
+}
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const mother = people.filter(mom => mom.name === person.motherName)[0];
+  const father = people.filter(dad => dad.name === person.fatherName)[0];
+  const { slug } = useParams();
+  const activeSlug = slug;
+
+  const baba = () => {
+    if (mother) {
+      return <Link to={`/people/${mother?.slug}`} className={classNames({ 'has-text-danger': mother?.sex === 'f' })}>{mother?.name}</Link>;
+    }
+
+    if (person.motherName) {
+      return person.motherName;
+    }
+
+    return '-';
+  };
+
+  const dada = () => {
+    if (father) {
+      return <Link to={`/people/${father?.slug}`}>{father?.name}</Link>;
```

Comment:
```suggestion
      return <Link to={`/people/${father.slug}`}>{father.name}</Link>;
```
you render it only when father exists